lfs/hook.go

```diff
@@ -22,6 +27,15 @@ type Hook struct {
 	Upgradeables []string
 }
 
+// NewStandardHook creates a new hook using the template script calling 'git lfs theType'
+func NewStandardHook(theType string, upgradeables []string) *Hook {
+	return &Hook{
+		Type:         theType,
+		Contents:     strings.Replace(hookBaseContent, "__CMD__", theType, -1),
```

Comment:
I'd prefer to use a `text/template` instead of string replacement, since the later is a common Go convention and is well understood by looking at the `hookBaseContent` variable alone.